packages/entity-list/src/containers/TableContainer.js

```diff
@@ -34,7 +34,8 @@ const mapStateToProps = (state, props) => ({
   positions: state.preferences.positions,
   widths: state.preferences.widths,
   markable: state.list.entityModel.markable && state.list.formDefinition.markable,
-  disablePreferencesMenu: state.list.disablePreferencesMenu
+  disablePreferencesMenu: state.list.disablePreferencesMenu,
+  scrollBehaviour: state.entityList.scrollBehaviour
```

Comment:
Side note (to discuss someday):
Is scrollBehaviour intended to change, once initialised? If not, I'm not sure if we should copy input params like this from `state.input` to another part of the state, or just read it from `state.input` when needed (like we do it a few lines above with the `navigationStrategy`). I think we don't follow a clear guideline here right now.

Reply:
Yes, good catch!
I wanted to test it within Storybook. That's why I added it here so that I could change the property there directly. 
But I was not sure if this is too good since I just added it for testing purposes.